src/lib/y2partitioner/widgets/blk_table.rb

```diff
@@ -0,0 +1,91 @@
+require "yast"
+
+require "cwm/table"
+
+require "y2partitioner/icons"
+
+module Y2Partitioner
+  module Widgets
+    # Table widget to represent given list of Block Devices.
+    class BlkTable
```

Comment:
IMHO abbreviations are acceptable for local variables only. API should use full names unless they are way too long.